packages/amplify-provider-awscloudformation/src/utils/progress-bar-helpers.ts

```diff
@@ -0,0 +1,81 @@
+import columnify from 'columnify';
+
+const COLUMNIFY_WIDTH = 30;
+
+export const CFN_SUCCESS_STATUS = [
+  'UPDATE_COMPLETE', 'CREATE_COMPLETE', 'DELETE_COMPLETE',
+  'DELETE_SKIPPED', 'UPDATE_ROLLBACK_COMPLETE', 'ROLLBACK_COMPLETE',
+];
+export const CNF_ERROR_STATUS = [
+  'CREATE_FAILED', 'DELETE_FAILED', 'UPDATE_FAILED',
+  'UPDATE_ROLLBACK_FAILED', 'ROLLBACK_FAILED',
+];
+
+type ItemPayload = {
```

Comment:
What is item?

Reply:
I found this is a duplicate of the other ItemPayload; and should probably be imported from the 'amplify-prompts' package instead of being re-declared. This code used to live inside of aws-utils/aws-cfn-progress-formatter.ts like this screenshot:

![image](https://user-images.githubusercontent.com/110861985/193143745-e629ccfa-1cf9-4726-93ed-ee7b8e5ce312.png)

Reply:
yeah, duplication isn't great (unless we have good reason like not exposing public api surface unnecessarily).
However I'm hoping to prompt discussion about what those properties really represent. `Item` is vague term, not far away from `object`.

Reply:
I fixed the import and added a backlog task to refactor ItemPayload to a more descriptive name.